packages/config/src/types.ts

```diff
@@ -915,6 +915,7 @@ export interface ProjectScalingRisks {
 
 export interface ProjectTvlInfo {
   associatedTokens: string[]
+  escrows: ProjectEscrow[]
```

Comment:
`ProjectTvlInfo` goes to the frontend. We want it to be light. Maybe a list of addresses if this is really needed. But you have this stuff in ProjectTvlConfig